roles/device_roles/germcaproto/tasks/main.yml

```
@@ -19,3 +19,32 @@
     cmd: "{{ ioc.pixi_executable_path | default(device_roles_germcaproto_pixi_executable_path) }} add 'python{{ ioc.python_version | default(device_roles_germcaproto_python_version) }}'"
     chdir: "{{ deploy_ioc_ioc_directory }}"
   changed_when: true
+
+- name: Add hextools dependency
+  ansible.builtin.command:
+    # yamllint disable-line rule:line-length
+    cmd: "{{ ioc.pixi_executable_path | default(device_roles_germcaproto_pixi_executable_path) }} add hextools"
```

Need to install this from PyPI like the original role has. Also, change the `git_revision` Ansible variable to use a PyPI version specifier (like how we did for Python).
```suggestion
    cmd: "{{ ioc.pixi_executable_path | default(device_roles_germcaproto_pixi_executable_path) }} add --pypi hextools"
```